app/views/budgets/show.html.erb

```diff
@@ -1,4 +1,6 @@
-<h1><%= t(".title") %></h1>
-<ul>
+<div class="box">
+  <h3><%= t(".title") %></h3>
   <%= render "budget" %>
-</ul>
+  <div class="button"><%= link_to t(".edit"), edit_budgets_path %>
+  <%= link_to t(".delete"), budgets_path, method: :delete %></div>
```

Comment:
A suggestion to clean the view up by using different html tags on different lines:
```
<div>
  <link>
  <link>
</div>
```

Reply:
Done in sha: a2fa585